facebook-instant-articles.php

```diff
@@ -404,4 +404,44 @@ function ia_markup_version( ) {
 	add_action( 'wp', 'ia_markup_version' );
 
 	Instant_Articles_Wizard::init();
+
+	function rescrape_article( $post_id, $post ) {
+		$adapter = new Instant_Articles_Post( $post );
+		$old_slugs = get_post_meta( $post_id, '_wp_old_slug' );
+		if ( $adapter->should_submit_post() ) {
+			try {
+				$client = Facebook\HttpClients\HttpClientsFactory::createHttpClient( null );
+				$url_encoded = urlencode($adapter->get_canonical_url());
+				$client->send(
+					"https://graph.facebook.com/?id=$url_encoded&scrape=true",
+					'POST',
+					'',
+					array(),
+					60
+				);
+				foreach ( $old_slugs as $slug ) {
+					$clone_post = clone $post;
```

Comment:
should this clonning really be inside the foreach? since only the post name is getting changed, maybe you can clone it once, outside, if at all.